cashu/wallet/cli/cli.py

```diff
@@ -81,31 +108,40 @@ def cli(ctx: Context, host: str, walletname: str):
     ctx.ensure_object(dict)
     ctx.obj["HOST"] = host or settings.mint_url
     ctx.obj["WALLET_NAME"] = walletname
-    wallet = Wallet(
-        ctx.obj["HOST"], os.path.join(settings.cashu_dir, walletname), name=walletname
-    )
+
+    db_path = os.path.join(settings.cashu_dir, walletname)
+    # loop = asyncio.new_event_loop()
```

Comment:
deprecated? also below.

Reply:
Yes, deprected because `cli` is now `async` :)